resources/wikia/libraries/jquery/loadLibrary/jquery.wikia.loadExternalLibrary.js

```diff
@@ -64,7 +64,10 @@
 				cookie: true,
 				version: 'v2.1'
 			});
-			// resolve after FB has finished inititalizing
+
+			// show facebook login button
+			$('.sso-login').removeClass('hidden');
```

Comment:
This looks like a nice alternative to the previous method. We'll be changing it again with https://wikia-inc.atlassian.net/browse/SOC-240 but this is an improvement for now.